rancher/charts
# AlertDrivers: Show General tab first, with SMS enabled by default

## Issue: https://github.com/rancher/rancher/issues/40735

## Problem
When installing the Alerting Drivers chart, the default configuration enables zero drivers, which is invalid and causes installation failure. Since the tab containing these settings is not selected by default, they was often missed by new users, causing minor confusion and overall friction.

## Solution
This moves the "General" tab so that it displays first in the list, as it contains the options most users are likely to want to tweak. For parity with earlier versions, and to ensure that the default configuration is valid, this tab now pre-selects "Enable SMS."

## Testing
Reproduction steps in https://github.com/rancher/rancher/issues/40735 are valid.

## Engineering Testing
### Manual Testing
Install the Alerting Drivers chart, version `102.0.0`. During the installation process, you should now see this updated UI:

![alertingdrivers_sms_default](https://user-images.githubusercontent.com/122321752/223789658-ac875607-b0b5-4347-864b-633551563d4d.png)

If no changes are made, the chart should install successfully. (If you manually deselect both drivers, an installation error is normal and expected.)
